setup/local-preview.md

```diff
@@ -0,0 +1,59 @@
+---
+title: "Local Preview"
+description: Setup a Coder deployment locally for testing.
+---
+
+Coder is normally deployed onto Kubernetes clusters, but that can be overkill
+for simply previewing the platform. To solve this, Coder provides a script to
+test out the platform locally using [Docker](https://www.docker.com/) and
+[kind](https://kind.sigs.k8s.io/)!
+
+Since it runs everything inside Docker, teardown is super simple and nothing is
+left on your machine.
+
+Local preview currently only supports Mac and Linux.
+
+## Installing
+
+In a Mac or Linux environment, ensure that the following tools are installed:
+
+1. `kubectl`: [https://kubernetes.io/docs/tasks/tools/install-kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl)
+1. `helm`: [https://helm.sh/docs/intro/install](https://helm.sh/docs/intro/install)
+1. `kind`: [https://kind.sigs.k8s.io/docs/user/quick-start/#installation](https://kind.sigs.k8s.io/docs/user/quick-start/#installation)
+
+Then, use the following script to install Coder:
+
+```bash
+curl -fsSL https://try.coder.com | PORT="8080" sh
```

Comment:
I'm assuming yes for this, but has this script been verified with mac's default old bash version? I ran it remotely earlier, but I didn't check on my MacBook. I can check tomorrow if it hasn't been tested

Reply:
I don't believe I'm using anything special so it's unlikely that it wouldn't work, but definitely wouldn't hurt to give it a try.

Reply:
It took a while to spin up the ingress, and then it failed with:
```
error: timed out waiting for the condition on pods/timescale-0
```

Reply:
timed out when retrying as well :(

Reply:
@coadler Do we have a solution for this? I was trying this out while editing and got the same time out error

Reply:
@khorne3 sorry, everyone in the office (both mac/linux) is able to run it fine so I haven't been able to figure out what exactly you're running into. While it's waiting for `timescale-0` to come up, could you potentially send me some logs?

* `kubectl describe pod timescale-0` 
* `kubectl logs timecale-0`

should help figure out what's going wrong.

Reply:
Hey @coadler here's the [output](https://docs.google.com/document/d/1ruxNQmnBdvFWi0KsNueOEq9PdLOtXAeWw2BrnI0LHlw/edit?usp=sharing) I'm getting; please let me know if I can provide anything else

Reply:
Thanks! I think I see why it's not coming up. I'll get an update pushed out and have you try again in a bit.

Reply:
@khorne3 try giving it another go

Reply:
@coadler Worked this time around. Thanks!